packages/matter.js/README.md

```diff
@@ -114,6 +114,8 @@ To have Typescript and your IDE know all the relevant exported functionality you
 }
 ```
 
+Additionally, you need to use at least `"strictNullChecks": true` or better for code quality `"strict": true` to make sure that all types are correctly determined.
```

Comment:
I think this is not necessary now but maybe we could make use of "strict: true" a recommendation.